concourse/steps/update_component_deps.mako

```diff
@@ -101,16 +101,6 @@ pull_request_util = github.util.PullRequestUtil(
     github_cfg=github_cfg,
 )
 
-## hack / workaround: rebase to workaround concourse sometimes not refreshing git-resource
-git_helper = gitutil.GitHelper(
-    repo=REPO_ROOT,
-    github_cfg=github_cfg,
-    github_repo_path=f'{REPO_OWNER}/{REPO_NAME}',
-)
-git_helper.rebase(
-    commit_ish=REPO_BRANCH,
-)
-
```

Comment:
as discussed: I am fine w/ simply removing rebase-code. however, we might trigger resource-check (to increase chance for next run to succeed)

Reply:
Yes, triggering resource-check seems like a good approach.
However, I still have to check auto-merge behaviour upon merge conflict, especially error handling. That's why PR is still on draft :-)